lib/metadata_gen.js

```diff
@@ -40,13 +40,8 @@ class MetadataGenerator {
       '' // creates final EOL
     ];
 
-    return meta.join(EOL);
+    return meta.join('\n');
```

Comment:
Why not sticking to EOL for cross-OS compatibility?

Reply:
@Tiriel This will get written to a file, better leave it as `\n`

Reply:
Thanks!